document-service/documents-ui/src/components/documentsTable/index.vue

```diff
@@ -20,28 +29,142 @@ const openDocumentRecord = (searchResult: DocumentInfoIF) => {
+            </UTooltip>
+            </div>
+            <UDivider class="my-3 w-full"/>
+            <div class="h-8">
+              <UInput
+                class="w-full px-2 font-light"
+                size="md"
+                :placeholder="column.label"
+              />
+            </div>
+          </div>
+        </template>
+        <template #consumerIdentifier-header = "{column}">
+          <div>
+            <div class="flex align-center px-2">
+              {{ column.label }}
+              <UTooltip
+                :popper="{ placement: 'top', arrow: true }"
+                :text="column.tooltipText"
+                class="w-20"
+              >
+              <UIcon
+                name="i-mdi-information-outline"
+                class="font-bold w-5 h-5 ml-2"
+              />
+            </UTooltip>
+            </div>
+            <UDivider class="my-3 w-full"/>
+            <div class="h-8">
+              <UInput
+                class="w-full px-2 font-light"
+                size="md"
+                :placeholder="column.label"
+              />
+            </div>
+          </div>
+        </template>
+        <template #documentURL-header = "{column}">
+          <div>
+            <div class="px-2">
+              {{ column.label }}
+            </div>
+            <UDivider class="my-3 w-full"/>
+            <div class="h-8">
+              <UInput
+                class="w-full px-2 font-light"
+                size="md"
+                :placeholder="column.label"
+              />
+            </div>
+          </div>
+        </template>
+        <template #consumerFilingDateTime-header = "{column}">
+          <div class="px-2">
+            {{ column.label }}
+          </div>
+          <UDivider class="my-3"/>
+          <div>
+          <div class="h-8">
+            <InputDatePicker
+              v-model="searchDateRange"
+              class="w-full px-2 font-light"
+              is-ranged-picker
+              is-left-bar
+              size="md"
+            />
+          </div>
+        </div>
+        </template>
+        <template #documentTypeDescription-header = "{column}">
+          <div class="px-2">
+            {{ column.label }}
+          </div>
+          <UDivider class="my-3"/>
+          <div>
+          <div class="h-8">
+            <USelectMenu
+                v-model="searchDocumentClass"
+                class="px-2 font-light"
+                select-class="text-gray-700"
+                :options="documentTypes"
+                value-attribute="class"
+                option-attribute="description"
+                size="md"
+                :placeholder="column.label"
+              />
+          </div>
+        </div>
+        </template>
+        <template #actions-header = "{column}">
+          <div class="px-2">
+            {{ column.label }}
+          </div>
+          <UDivider class="my-3"/>
+          <div>
+          <div class="h-8">
+          </div>
```

Comment:
Is this for spacing? If so you can also `<div class="h-8" />` or use a NuxtUi native component for spacing. either or.